pkg/sippyserver/server.go

```diff
@@ -88,6 +89,14 @@ func (s *Server) RefreshData() {
 	klog.Infof("Refresh complete")
 }
 
+func (s *Server) defaultHandler(w http.ResponseWriter, req *http.Request) {
+	if req.URL.Path == "/" {
+		s.printHtmlReport(w, req)
+	} else {
+		generichtml.PrintStatusMessage(w, 404, "Page not found.")
```

Comment:
Instead of `404`, refer to the constant from the `http` package, `http.StatusNotFound`.

Reply:
Oops, fixed.